src/OpenIddict.Core/OpenIddictBuilder.cs

```diff
@@ -431,7 +432,7 @@ public class OpenIddictBuilder {
         /// </summary>
         /// <returns>The <see cref="OpenIddictBuilder"/>.</returns>
         public virtual OpenIddictBuilder UseJsonWebTokens() {
-            return Configure(options => options.UseJwtTokens());
+            return Configure(options => options.AccessTokenHandler = new JwtSecurityTokenHandler());
```

Comment:
JWT is still supported for access tokens, but you have to use the long form: `options.AccessTokenHandler = new JwtSecurityTokenHandler()`, which is exactly what OpenIddict's extension does.